app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="maximum_risk_score">6</integer>
```

Comment:
Business logic constants are not stored in the resources. Let's put it into the `MenuFragment`

Reply:
I thought you would like this -- but sure